templates/zerver/help/import-data-from-slack.md

```diff
@@ -42,3 +42,6 @@ If the users are not from the same organization, you should change your organiza
   by Zulip's backend markdown.
 - The "joined #channel_name" messages. They are intentionally removed because
   they are spammy.
+- Zulip does not support default channels which can be unsubscribed from, but
```

Comment:
Which can't be unsubscribed from, like Slack's '#general' channel.

Reply:
My bad. Corrected!